subsys/bluetooth/host/hci_core.c

```diff
@@ -21,12 +21,14 @@
 
 #define BT_DBG_ENABLED IS_ENABLED(CONFIG_BLUETOOTH_DEBUG_HCI_CORE)
 #include <bluetooth/log.h>
+#include <common/log.h>
```

Comment:
Then I like the idea of moving log.h to subsys/bluetooth/common and using that from drivers/bluetooth yes. That way we avoid having private stuff in include/. I will push this